example/example.cpp

```diff
@@ -143,7 +143,7 @@ void async_example()
 {
     // Default thread pool settings can be modified *before* creating the async logger:
     // spdlog::init_thread_pool(32768, 1); // queue with max 32k items 1 backing thread.
-    auto async_file = spdlog::basic_logger_mt<spdlog::async_factory>("async_file_logger", "logs/async_log.txt");
+    auto async_file = spdlog::basic_logger_mt<spdlog::async_factory>("async_file_logger", SPDLOG_FILENAME_T("logs/async_log.txt"));
```

Comment:
While correct, I think the example should be without this SPDLOG_FILENAME_T macro. It is only needed in very special cases (windows + wchar support).